plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcPageSink.java

```diff
@@ -186,22 +177,8 @@ else if (javaType == Slice.class) {
         return completedFuture(ImmutableList.of());
     }
 
-    @SuppressWarnings("unused")
     @Override
-    public void abort()
-    {
-        // rollback and close
-        try (Connection connection = this.connection;
```

Comment:
restore closing as there is no explicit `close()` method to close the connection and statement.